hooks/utils.py

```diff
@@ -290,6 +247,10 @@ def install_builtin_server():
     requirements = os.path.join(CURRENT_DIR, 'server-requirements.pip')
     # Install the builtin server dependencies avoiding to download requirements
     # from the network.
+    # XXX frankban: this pip installation here implicitly depends on juju-gui
+    # dependencies to be installed. In essence, this function does not fetch
+    # dependencies from the network only incidentally, because setup_gui() has
+    # been already called in the unit.
```

Comment:
Is this an XXX? We usually use those for things we need to change soon. This seems like a regular comment--definitely good to know, but not something that's a quick hack or a problem.